moto/cloudfront/exceptions.py

```diff
@@ -15,10 +16,11 @@
 
 class CloudFrontException(RESTError):
     code = 400
+    extended_templates = {"cferror": EXCEPTION_RESPONSE}
+    env = Environment(loader=DictLoader(RESTError.templates | extended_templates))
```

Comment:
The `|`-operator is not supported in Python 3.7, as far as I know. I'll kick off the linter anyway, but I expect that to fail.

Reply:
good catch,  this is a 3.9 feature it turns out